source/StepBro.Core/IPCPipe/IPCPipe.cs

```diff
@@ -1,18 +1,15 @@
 ﻿using System;
 using System.Collections.Concurrent;
-using System.Collections.Generic;
-using System.ComponentModel.DataAnnotations;
-using System.Diagnostics;
-using System.IO;
 using System.IO.Pipes;
+using System.IO;
 using System.Security.Principal;
 using System.Text;
 using System.Text.Json;
 using System.Threading;
 
-namespace StepBro
+namespace StepBro.Core.IPCPipe
```

Comment:
Class and namespace with same name is not a good idea. Maybe namespace IPC and class Pipe.